client/src/components/History/CurrentHistory/HistoryOperations/SelectionOperations.vue

```diff
@@ -10,21 +10,21 @@
                 </span>
             </template>
             <b-dropdown-text>
-                <span v-localize>With {{ numSelected }} selected...</span>
+                <span v-localize data-test-id="selected-count">With {{ numSelected }} selected...</span>
```

Comment:
This is amazing, thank you for the fix and all the tests, this is exactly what the new history needs. I have a minor suggestion here, I suggest we use `data-description` instead of `data-test-id`.

Reply:
Done in [f4251e2](https://github.com/galaxyproject/galaxy/pull/13911/commits/f4251e2fbbec623b2abaaa775fbdaa42d4a9c913)